tests/test_entities.py

```diff
@@ -100,6 +101,7 @@ def test_self_path_with_which(self, entity, path, which):
     @data(
         (entities.ConfigTemplate, '/config_templates', 'build_pxe_default'),
         (entities.ConfigTemplate, '/config_templates', 'revision'),
+        (entities.RHCIDeployment, '/deployments', 'deploy'),
```

Comment:
Yep. Just drop this line, and add `(entities.RHCIDeployment, '/deployments')` to [`test_path_without_which`](https://github.com/SatelliteQE/nailgun/blob/master/tests/test_entities.py#L41).

Yes, this is a PITA. But I've been bitten by omitting these tests in the past.